plugins/inputs/cloud_pubsub_push/pubsub_push.go

```diff
@@ -169,9 +169,13 @@ func (p *PubSubPush) Start(acc telegraf.Accumulator) error {
 	go func() {
 		defer p.wg.Done()
 		if tlsConf != nil {
-			p.server.ListenAndServeTLS("", "")
+			if err := p.server.ListenAndServeTLS("", ""); err != nil {
+				p.Log.Errorf("listening and serving TLS failed: %v", err)
+			}
 		} else {
-			p.server.ListenAndServe()
+			if err := p.server.ListenAndServe(); err != nil {
```

Comment:
`ListenAndServeTLS always returns a non-nil error. After Shutdown or Close, the returned error is ErrServerClosed.` -> Maybe these errors should be printed only when `err != ErrServerClosed`

Reply:
Very good point. Will change.